index.html

```diff
@@ -67,5 +67,130 @@ <h5 class="small-text">I can help you build a product , feature or website Look
             </li>             
      </ul>               
     </section>
+    <section class="works">
+        <div class="title">
+           <h2>My Recent Works</h2>
+           <div class="line"></div>
+        </div>
+        <div class="sample">
+           <img src="./assets/images/works-section.png" alt="works">
+           <h3>Multi-Post Stories</h3>
+           <p class="works-para">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+           </p>
+           <ul class="tags">
+            <li class="tag-item">CSS</li>
+            <li class="tag-item">HTML</li>
+            <li class="tag-item">Bootstrap</li>
+            <li class="tag-item">Ruby</li>
+           </ul>
+           <button type="button">See Project</button>
+        </div>
+        <div class="works-grid">
+           <div class="work-item">
+            <div class="work-item-content">
+                <h3>Multi-Post Stories</h3>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+                <ul class="tags">
+                    <li class="tag-item">CSS</li>
+                    <li class="tag-item">HTML</li>
+                    <li class="tag-item">Bootstrap</li>
+                    <li class="tag-item">Ruby</li>
+                </ul>
```

Comment:
- [ ] I have realized that the list elements of this section are overlapping because it has one extra list element that should not be here. Kindly remove HTML from the list elements.

**This is how I am seeing your design right now:**

![image](https://user-images.githubusercontent.com/86318284/185756533-14867167-48a2-45b7-ad37-cd7ad8c90e7c.png)


**This is how it is supposed to look like:**

![image](https://user-images.githubusercontent.com/86318284/185756559-b6109ed3-427a-4411-bd3e-42f2874e3f45.png)

**Kindly make this change for all your individual projects**